### PR TITLE
Module: Lich Pet Summon fix

### PR DESCRIPTION
**Pull request by Argavyon**
Makes the overly wide petlist for liches with summon zombie+skeleton+FM+necro more eye-friendly.

Goes from this
Screenshot_3
To this
Screenshot_4

### CODE REVIEW

File: nexus-tweaks.user.js [hunk 1]
```diff
@@ -1144,6 +1144,43 @@ promiseList.push((async () => {
...
+
+  const lichDelight = () => {
+    'use strict';
+
+    const petTableTable = document.querySelector('.petTable');
+    if (!petTableTable) return;
+
+    // Pets summon buttons
+    const petSummons = petTableTable.querySelector('tbody').querySelector('table.summonsButtons>tbody');
+    const FM = petSummons.querySelector('input[value="Fossil Monstrosity"]');
```
Comment (AnneTrue) [inline nexus-tweaks.user.js:1164]: Style-wise it's consistent as "fossilMonstrosity", rather than all-caps.

File: nexus-tweaks.user.js [hunk 2]
```diff
@@ -1144,6 +1144,43 @@ promiseList.push((async () => {
...
+    const necro = petSummons.querySelector('input[value="Necrophage"]');
+
+    if (FM) {
+      FM.nextSibling.value = 'Fossil Monstrosity (3 Skeletons) '
```
Comment (AnneTrue) [inline nexus-tweaks.user.js:1168]: What's the purpose of this? Are you cutting off the MP cost?

File: nexus-tweaks.user.js [hunk 3]
```diff
@@ -1144,6 +1144,43 @@ promiseList.push((async () => {
...
+      if (petSummons.children.length < 3) petSummons.appendChild(document.createElement('tr'));
+      petSummons.children[2].insertBefore(FM.parentNode.parentNode, petSummons.children[2].firstChild);
```
Comment (AnneTrue) [inline nexus-tweaks.user.js:1170]: When you're manipulating DOM elements with relative positions like this, it helps a lot to document what you expect those elements to be.

**Review by AnneTrue — CHANGES_REQUESTED** [review]
Two overall questions:
- Are the devs fixing this directly? This is very much a problem they should fix.
- Is this problem actually specific to liches because they have the dual titans, or will other classes benefit from a more generic approach?

Besides that, needs the usual check-off of pre-merge items such as README documentation and version bumps.

**Comment by Argavyon** [comment]
Version bumping isn't trivial from my side, as I can't tell what will be merged first. (In particular, whether this will be merged before the badge classifier).
This would be an even larger issue if someone else was sending PRs.

**Pull request closed without merging**